### ISSUE TITLE
Search: Design search bar with filters

### ISSUE THREAD

**Issue by He1DAr** [hirosystems/explorer#1669]

**Comment by eugeniadigon** [comment]
I don't think we should have filters on the search bar (other than "Category") because:

- If there are more than 5 search results, we should link to the Full Search Results page to facilitate easier filtering and parsing of the results
- If there are less than 5 results, filtering becomes redundant
